### PR TITLE
watchdog can be disabled by a flag

### PR DESCRIPTION
**Pull request by starsy**

### CODE REVIEW

File: scripts/main-run [hunk 1]
```diff
@@ -55,16 +55,21 @@ $FIREWALLA_HOME/scripts/main-start
...
-sleep 1
-$FIREWALLA_HOME/scripts/fire-ping.sh
-sudo service watchdog stop
-echo "WatchDog Stopped"
-$FIREWALLA_HOME/scripts/fire-ping.sh
-sleep 3 
-sudo service watchdog start
-echo "WatchDog Started"
+if [ -e /home/pi/.firewalla/config/disable_watchdog ]; then
+    logger "FIREWALLA:WATCHDOG:DISABLED"
```
Comment (MelvinTo) [inline scripts/main-run]: should call "sudo systemctl stop watchdog" when the disable flag exists, in case it's already started.

Comment (starsy) [reply scripts/main-run]: added

**Comment by jerrchen** [comment]
Ticket #388, we have one customer's router gateway ping is not stable.  Which causes fire-ping to fail.  Is it possible to add a flag somewhere, once that's enabled, fire-ping will not reboot.   The problem likely is only with pings

**Comment by MelvinTo** [comment]
this is a separate feature, will merge this first.

**Pull request merged**